src/addons/Portal/Portal.js

```diff
@@ -363,6 +363,14 @@ class Portal extends Component {
     )
 
     this.portal = this.node.firstElementChild
+    // don't take focus away from portals that close on blur
+    if (!closeOnTriggerBlur) {
+      this.triggerNode = document.activeElement
+      this.portal.setAttribute('tabindex', '-1')
+      this.portal.style.outline = 'none'
+      // wait a tick for things like popups which need to calculate where the popup shows up
+      setTimeout(() => this.portal && this.portal.focus())
```

Comment:
Technically this code is being called by either componentDidMount or componentDidUpdate depending on when 'open' is set to true which is after the render method. Or are you saying you'd like me to make a named helper function that's called here? There's no later life cycle function I can use to trigger this behavior